Expensify/App
# [User Settings] Implement Settings Menu Selectors

### Details
- Created a new MenuItem component
- Set up subroutes for the settings page

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify.cash/issues/1493

### Tests
1. Click into Settings page from HomePage
2. Click each menu item and confirm subroute renders

### Tested On


### Video Walkthrough

https://user-images.githubusercontent.com/12075600/109905485-bbcf8000-7c53-11eb-869e-0f1dd7d89c97.mov



#### Web

![Screen Shot 2021-03-03 at 6 56 01 PM](https://user-images.githubusercontent.com/12075600/109905526-c722ab80-7c53-11eb-89e0-0d2d640ea320.png)

#### Mobile Web

<img width="494" alt="Screen Shot 2021-03-03 at 7 00 30 PM" src="https://user-images.githubusercontent.com/12075600/109905546-cf7ae680-7c53-11eb-99fc-ae0e88812f7f.png">

#### Desktop

<img width="1312" alt="Screen Shot 2021-03-03 at 6 55 10 PM" src="https://user-images.githubusercontent.com/12075600/109905577-d73a8b00-7c53-11eb-97dd-05fbb36ad87a.png">

#### iOS

<img width="494" alt="Screen Shot 2021-03-03 at 7 02 42 PM" src="https://user-images.githubusercontent.com/12075600/109905595-df92c600-7c53-11eb-8ec2-bcf09c24dbbe.png">

#### Android

<img width="464" alt="Screen Shot 2021-03-03 at 6 58 52 PM" src="https://user-images.githubusercontent.com/12075600/109905623-ea4d5b00-7c53-11eb-940e-3ab2c8e82033.png">
